app/views/teams/_solved_challenges_table.html.haml

```diff
@@ -1,14 +1,16 @@
 %table.table.table-bordered.table-striped
   %thead
     %tr
-      %th #
-      %th= t('teams.summary.solved_challenges_table.challenge_header')
-      %th= t('teams.summary.solved_challenges_table.points_header')
-      %th= t('teams.summary.solved_challenges_table.when_header')
+      %th Challenge
+      %th Points
+      %th When
   %tbody
     - @team.solved_challenges.each_with_index do |s, i|
-      %tr
-        %td= i + 1
+      %tr{i > 4 ? {class: "hide"} : {}}
```

Comment:
Shouldn't this be `i > 5` like it is below? Otherwise we're adding the `hide` class on the 5th challenge but not adding the toggle button until the 6th.

Reply:
The `i > 4` refers to the index of the challenge in the list which gives me 5 rows.